src/components/Main/Main.js

```diff
@@ -24,11 +27,24 @@ export class MainComponent {
     );
   }
 
+  addEventListeners() {
+    this.sidebarList.addEventListener("click", this.handleListClick.bind(this));
+  }
+
+  handleListClick(e) {
+    e.preventDefault();
+    if (e.target.closest(".main__about-link")) {
+      this.activeView = new AboutComponent(this.mainPoint);
```

Comment:
you could initialize you component in `mountChildren` and then just call `mount` here. There is no need for active view currently